Portfolio/index.css

```diff
@@ -1,15 +1,20 @@
-/* Write your code below for PORTFOLIO */
 * {
     margin: 0px;
     padding: 0px;
     font-family: "Revalia", sans-serif;
     color: #00D8FF;
+    box-sizing: border-box;
 
 }
 
-.container {
-    width: 100vw;
+body {
     height: 1200vh;

```

Comment:
The .about-article class is not properly defined. It should have properties inside the curly braces.